#name of object that needs to be queried again
ObjectState[] object_states
---
int32 SUCCESS=0
int32 UNKNOWN_OBJECT=1
#response fields
int32 error_code
